File: src/main/java/io/ctl/portal/challenge/model/Person.java
package io.ctl.portal.challenge.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Person {
    private int id;
    private String firstName;
    private String lastName;
    private int age;
}

File: src/main/javascript/challenge/src/Person/Person.js
import React from 'react';

const person = (props) => {
    return (
        <p>
            <b>First Name: </b>{props.firstName}&nbsp;
            <b>Last Name: </b>{props.lastName}&nbsp;
            <b>Age: </b>{props.age}&nbsp;
        </p>
    );
}

export default person;